src/main/java/pl/edu/pk/cosmo/cookbookbackend/service/impl/AccountServiceImpl.java

```diff
@@ -0,0 +1,73 @@
+package pl.edu.pk.cosmo.cookbookbackend.service.impl;
+
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.stereotype.Service;
+import pl.edu.pk.cosmo.cookbookbackend.converter.AccountConverter;
+import pl.edu.pk.cosmo.cookbookbackend.repository.Entity.Account;
+import pl.edu.pk.cosmo.cookbookbackend.repository.interfaces.AccountRepository;
+import pl.edu.pk.cosmo.cookbookbackend.service.dto.AccountDTO;
+import pl.edu.pk.cosmo.cookbookbackend.service.exception.AlreadyExistsException;
+import pl.edu.pk.cosmo.cookbookbackend.service.exception.NoAccountException;
+import pl.edu.pk.cosmo.cookbookbackend.service.interfaces.AccountService;
+
+import javax.transaction.Transactional;
+import javax.validation.constraints.NotNull;
+
+@Service
+@Scope(proxyMode = ScopedProxyMode.INTERFACES)
```

Comment:
Nie do końca rozumiem dlaczego użyłeś ten proxyMode i @Transactional. Czy jest to tutaj potrzebne?

Reply:
@Transactional jest potrzebne do zmiany hasła, lecz proxyMode nie był